utility-evaluator/src/main/java/eu/nebulous/utilityevaluator/communication/sal/NodeCandidatesFetchingService.java

```diff
@@ -49,12 +49,20 @@ public List<NodeCandidate> getNodeCandidatesViaMiddleware(Application app, Strin
             e.printStackTrace();
         }*/
 
+        // rudi, 2024-09-25: Note that we send an empty reuqirements list, so
+        // we get all known node candidates, not only the ones required by
```

Comment:
Yes, it is done to make the regression over the price & Node Candidates parameters. It can be improved but this behaviour is not incorrect